### ISSUE TITLE
[0.2.0] release 0.2.0 not working out of the box on AWS ECS (as replacement of 0.1.0)

### ISSUE THREAD

**Issue by giom-l** [provectus/kafka-ui#853]
**Final answer**
If you read that, please consider that the issue is not real. 
It was due to an healthcheck configuration that can't work OOTB with release > 0.2.0 (as docker image does not contain `curl` command anymore.

**Describe the bug**
Well, that is surely due to some misconfiguration, but I can't figure out which one...
I currently have kafka-ui 0.1.0 running fine in AWS ECS.
I wanted to update a testing environment this morning with 0.2.0, but it fails...
All errors appear in JS console of my browser (I tested on firefox, chrome & brave, no differences)

Here are the error I get 

```
Refused to apply style from 'https://kafka-ui/static/css/2.f8a277e1.chunk.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

Failed to load resource: the server responded with a status of 404 ()  2.b8ce6301.chunk.js:1
Manifest: Line: 1, column: 1, Syntax error.
Refused to apply style from 'https://kafka-ui/static/css/main.7c3138c7.chunk.css' because its MIME type ('application/json') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

Is there any know configuration that I should apply ? 
I just changed the docker image to use, no extra change in ECS configuration.

**Set up**
The app is running on AWS ECS, with following configuration
```
{
  "ipcMode": null,
  "executionRoleArn": "arn:aws:iam::AccountID:role/kafka-ui",
  "containerDefinitions": [
    {
      "dnsSearchDomains": null,
      "environmentFiles": null,
      "logConfiguration": {
        "logDriver": "awslogs",
        "secretOptions": null,
        "options": {
          "awslogs-group": "/kafka-ui",
          "awslogs-region": "eu-west-1",
          "awslogs-stream-prefix": "kafka-ui"
        }
      },
      "entryPoint": null,
      "portMappings": [
        {
          "hostPort": 8080,
          "protocol": "tcp",
          "containerPort": 8080
        }
      ],
      "command": null,
      "linuxParameters": null,
      "cpu": 0,
      "environment": [
        {
          "name": "KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS",
          "value": "kafka1:9092,kafka2:9092,kafka3:9092"
        },
        {
          "name": "KAFKA_CLUSTERS_0_NAME",
          "value": "Kafka Dev"
        },
        {
          "name": "KAFKA_CLUSTERS_0_READONLY",
          "value": "true"
        },
        {
          "name": "KAFKA_CLUSTERS_1_BOOTSTRAPSERVERS",
          "value": "kafka1.logs:9092,kafka2.logs:9092,kafka3.logs:9092"
        },
        {
          "name": "KAFKA_CLUSTERS_1_NAME",
          "value": "Kafka Logs"
        },
        {
          "name": "KAFKA_CLUSTERS_1_READONLY",
          "value": "true"
        }
      ],
      "resourceRequirements": null,
      "ulimits": null,
      "dnsServers": null,
      "mountPoints": [],
      "workingDirectory": null,
      "secrets": null,
      "dockerSecurityOptions": null,
      "memory": null,
      "memoryReservation": null,
      "volumesFrom": [],
      "stopTimeout": null,
      "image": "provectuslabs/kafka-ui:0.2.0",
      "startTimeout": null,
      "firelensConfiguration": null,
      "dependsOn": null,
      "disableNetworking": null,
      "interactive": null,
      "healthCheck": {
        "retries": 3,
        "command": [
          "CMD-SHELL",
          "curl -f http://localhost:8080/ || exit 1"
        ],
        "timeout": 5,
        "interval": 30,
        "startPeriod": 60
      },
      "essential": true,
      "links": null,
      "hostname": null,
      "extraHosts": null,
      "pseudoTerminal": null,
      "user": null,
      "readonlyRootFilesystem": true,
      "dockerLabels": null,
      "systemControls": null,
      "privileged": null,
      "name": "kafka-ui"
    }
  ],
  "placementConstraints": [],
  "memory": "512",
  "taskRoleArn": "arn:aws:iam::AccountID:role/kafka-ui-task",
  "compatibilities": [
    "EC2",
    "FARGATE"
  ],
  "taskDefinitionArn": "arn:aws:ecs:eu-west-1:AccountID:task-definition/kafka-ui:19",
  "family": "kafka-ui",
  "requiresAttributes": [
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "com.amazonaws.ecs.capability.logging-driver.awslogs"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "ecs.capability.execution-role-awslogs"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "com.amazonaws.ecs.capability.docker-remote-api.1.19"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "com.amazonaws.ecs.capability.docker-remote-api.1.17"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "com.amazonaws.ecs.capability.task-iam-role"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "ecs.capability.container-health-check"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "com.amazonaws.ecs.capability.docker-remote-api.1.18"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "ecs.capability.task-eni"
    },
    {
      "targetId": null,
      "targetType": null,
      "value": null,
      "name": "com.amazonaws.ecs.capability.docker-remote-api.1.29"
    }
  ],
  "pidMode": null,
  "requiresCompatibilities": [
    "FARGATE"
  ],
  "networkMode": "awsvpc",
  "cpu": "256",
  "revision": 19,
  "status": "ACTIVE",
  "inferenceAccelerators": null,
  "proxyConfiguration": null,
  "volumes": []
}
```

**Steps to Reproduce**

1. 

**Expected behavior**
Kafka-ui 0.2.0 working as previous version is.

**Screenshots**
![image](https://user-images.githubusercontent.com/29728596/132188376-8a584281-553b-4fa6-b888-21b0bc796a49.png)

**Additional context**
(Add any other context about the problem here)

**Comment by RustamGimadiev** [comment]
Hi @giom-l, thank you for the issue, could you please share the full source code of the page? Tried to reproduce this issue locally, but was not able to do that. We have a demo stand on ECS and it is also working correctly with that version.

**Comment by giom-l** [comment]
Sure, 

It is is 
```


Line wrap
--
  | <!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="icon" href="./favicon.ico"/><meta name="viewport" content="width=device-width,initial-scale=1"/><link rel="apple-touch-icon" href="./logo192.png"/><link rel="manifest" href="./manifest.json"/><title>UI for Apache Kafka</title><script type="text/javascript">window.basePath=""</script><link href="/static/css/2.f8a277e1.chunk.css" rel="stylesheet"><link href="/static/css/main.7c3138c7.chunk.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div><script>!function(e){function r(r){for(var n,a,f=r[0],i=r[1],l=r[2],c=0,s=[];c<f.length;c++)a=f[c],Object.prototype.hasOwnProperty.call(o,a)&&o[a]&&s.push(o[a][0]),o[a]=0;for(n in i)Object.prototype.hasOwnProperty.call(i,n)&&(e[n]=i[n]);for(p&&p(r);s.length;)s.shift()();return u.push.apply(u,l\|\|[]),t()}function t(){for(var e,r=0;r<u.length;r++){for(var t=u[r],n=!0,f=1;f<t.length;f++){var i=t[f];0!==o[i]&&(n=!1)}n&&(u.splice(r--,1),e=a(a.s=t[0]))}return e}var n={},o={1:0},u=[];function a(r){if(n[r])return n[r].exports;var t=n[r]={i:r,l:!1,exports:{}};return e[r].call(t.exports,t,t.exports,a),t.l=!0,t.exports}a.m=e,a.c=n,a.d=function(e,r,t){a.o(e,r)\|\|Object.defineProperty(e,r,{enumerable:!0,get:t})},a.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},a.t=function(e,r){if(1&r&&(e=a(e)),8&r)return e;if(4&r&&"object"==typeof e&&e&&e.__esModule)return e;var t=Object.create(null);if(a.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:e}),2&r&&"string"!=typeof e)for(var n in e)a.d(t,n,function(r){return e[r]}.bind(null,n));return t},a.n=function(e){var r=e&&e.__esModule?function(){return e.default}:function(){return e};return a.d(r,"a",r),r},a.o=function(e,r){return Object.prototype.hasOwnProperty.call(e,r)},a.p="./";var f=this["webpackJsonpkafka-ui"]=this["webpackJsonpkafka-ui"]\|\|[],i=f.push.bind(f);f.push=r,f=f.slice();for(var l=0;l<f.length;l++)r(f[l]);var p=i;t()}([])</script><script src="/static/js/2.b8ce6301.chunk.js"></script><script src="/static/js/main.783a1f47.chunk.js"></script></body></html>


```

**Comment by giom-l** [comment]
I just noticed one more thing.
In my browser, the type for files that are not retrived seems to be wrong (I don't know if that is a default value or an error that lead to the failure)

![image](https://user-images.githubusercontent.com/29728596/132252312-4436d590-90d9-4f02-b922-6b64cc946348.png)

**Comment by giom-l** [comment]
Well, I could make this version (0.2.0) work in my EKS cluster... 
Still wondering what could be the issue that make the 0.1.0 work and the 0.2.0 fail in ECS (with exact same configuration)

**Comment by giom-l** [comment]
After some F5 hammering, I sometime get the page that display 0.2.0 version.
And I think I found the culprit.

My healthcheck is set as 
```
healthcheck = {
    command     = ["CMD-SHELL", "curl -f http://localhost:8080/ || exit 1"]
```
I set it like that as curl was available in `0.1.0` image (which was based on `openjdk:13`)
However it is not available in 0.2.0 anymore (which is now based on `adoptopenjdk/openjdk13:x86_64-alpine-jre-13.0.2_8`)

After removing the healthcheck, everything is back to normal.
I think I had this strange failures because the lapse time during which ECS redirect requests to it before it was terminated was very short (and so some resources were not served anymore).

So, last question : is there any health endpoint I could use to restore my healthcheck back ? :)

**Comment by germanosin** [comment]
@giom-l. Thanks for sharing the info. I'm happy that it's finally working fine for you.

**Comment by giom-l** [comment]
Hi @germanosin 
I tested the master branch, and it works :)
It took 58s to fetch all my consumer groups (I have 582 consumer on this cluster, but no error). 
However, the json size is not that big : is the fetch time mostly due to the response time from kafka ? 

Beside this issue (which is solved ), I noticed a call on github api. Is it expected ? 

Here is a picture of both response time and call on github api.
![image](https://user-images.githubusercontent.com/29728596/132342364-b1ebb4c8-580e-4770-b99a-faf6ea11816f.png)
